staging/src/k8s.io/client-go/tools/pager/pager.go

```diff
@@ -67,6 +80,21 @@ func New(fn ListPageFunc) *ListPager {
 	}
 }
 
+// New creates a new pager from the provided pager function using the default size
+// options. It will fall back to a full list if an expiration error is encountered
+// as a last resort.
+func NewNamed(name string, fn ListPageFunc, backoff wait.BackoffManager, retries int) *ListPager {
```

Comment:
nit: I think the doc comment here needs to be updated to reflect `NewNamed`'s functionality

Reply:
See revised PR.